en/docs/monitoring-and-insights/alerts-overview.md

```diff
@@ -0,0 +1,145 @@
+
+Configurable Parameters:
+
+- Metric: CPU or Memory.
+- Threshold: **mCPU** for CPU and **MiB** for Memory(e.g., 1000).
+- Period: Duration the threshold must be exceeded (e.g., 5 minutes).
+
+    !!! Tip
+        - **CPU**: mCPU (milliCPU) measures CPU usage in fractions of a core, where 1000m = 1 full core.
+        - **Memory**: MiB (Mebibyte) measures memory in binary units, where 1 MiB = 2^20^ bytes.
+
+### Logs alerts
+
+Logs alerts trigger notifications when a specific phrase appears frequently in your component logs. This helps to identify recurring issues or critical errors quickly, enabling faster troubleshooting.
+
+Configurable Parameters:
+
+- Search Phrase: Keyword or phrase (e.g., failed).
+- Count: Minimum occurrences to trigger the alert (e.g., 10).
+- Interval: Time window for counting occurrences (e.g., 5 minutes).
+
+### Build Failure alerts
+
+Build Failure alerts inform you if a build failure occurs for your component. This is essential for maintaining smooth development workflows, as it allows quick action to reduce downtime.
+
+### Status Code alerts
+
+Status Code alert triggers when your component returns specific HTTP error(s) (e.g., **403** Forbidden, **500** Internal Error). These alerts help to detect issues affecting your component’s availability or speed.
+
+Configurable Parameters:
+
+- Status Code: Error code or series (e.g., 400:Bad Request).
+- Count: Minimum occurrences (e.g., 5).
+- Interval: Time window (e.g., 5 minutes).
+
+!!! note
+    Currently Status Code alerts only supports api proxy component types.
+
+## Configure Alert
+
+If you are going to create an alert for the first time, follow the steps given below.
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/).
+2. Ensure you are in the correct organization where you have a project with the component to configure an alert.
+3. Navigate to the component by clicking on the project with the component to configure an alert.
+4. Click the component.
+
+    !!! info
+        make sure your component is deployed in order to make the alerts work.  
+
+5. In the Choreo Console left navigation menu, click **Observability**.
+6. In the left navigation menu on the **Observability** page, click **Alerts**. This opens the **Configure Alerts** pane by default.
+7. Click **Create Alert Rule** to create a new alert rule.
+
+    ![Create Alert Rule](../assets/img/monitoring-and-insights/alerts/alert-creation.png){.cInlineImage-full}
+
+    ![Create Alert Rule](../assets/img/monitoring-and-insights/alerts/alert-configuration.png){.cInlineImage-full}
+
+    This opens the **Alert Creation** page with the **Latency** alert type selected by default.
+
+8. Select the **[Alert Type](#supported-alert-types)** you want to create.
+9. Select the **Environment** you want to create the alert for.
+10. Select the **Deployment Track** as required.
+11. In the **Metric** field, select the required metric against which you want to evaluate the alert configuration.
```

Comment:
This field is not available for all alert types. Hence IMO, we can explain the common configurations (Emails, Environment etc..) and say configure the other fields specific to each alert type